source/sync/data-access-patterns/permissions.txt

```diff
@@ -360,6 +353,183 @@ change until the next session.
      write rules. Functions would not operate on a per-document basis.
    - You cannot store permissions information (such as "which document IDs may this user access?") in the user object.
 
+
+.. _flexible-sync-roles-json-reference:
+
+Flexible Sync Roles JSON Reference
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A **Role** object can specify a specific collection's role or the default role
+used for any unspecified collection. You define field-level permissions with a
```

Comment:
What do you mean by "any unspecified collection"? It implies, I think, that the default role doesn't apply to a specified collection...which I know isn't sensible but might not be clear to the reader. Would this be more clear:
"or to the default role applied to all collections"?